plextraktsync/plex/PlexApi.py

```diff
@@ -200,12 +200,20 @@ def history(self, m, device=False, account=False):
             yield h
 
     @retry()
-    def mark_watched(self, m):
-        m.markPlayed()
+    def mark_watched(self, m, is_discover=False):
+        if is_discover:
+            acc = self.account
+            acc.markPlayed(m)
```

Comment:
```suggestion
            self.account.markPlayed(m)
```

single-use temporary variable not needed. and the line length is not overflowing either

Reply:
this helps to understand what caused a crash. account call or markPlayed call.